stytch-auth/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "stytch-auth",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build --mode client && vite build",
```

Comment:
If you use `@cloudflare/vite-plugin` to build, I think you can run only `vite build`. Can you confirm it?